project2/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Crowned Article Card</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.typekit.net/xvl5rma.css">
+    <link rel="stylesheet" href="https://use.typekit.net/xvl5rma.css">
+    <link rel="stylesheet" href="https://use.typekit.net/xvl5rma.css">
+
+
+</head>
+
+<body>
+
+    <div class="card">
+
+        <div class="content">
+
+            <div class="img-wrapper">
+
+                <a href="#"><svg class="favorite" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 52.18">
+                        <defs>
+                            <style>
+                                .cls-2 {
+                                    fill: #fff
+                                }
+                            </style>
+                        </defs>
+                        <g id="Layer_2" data-name="Layer 2">
+                            <g id="Layer_1-2" data-name="Layer 1">
+                                <path
+                                    d="M25.07 49.59a4 4 0 01-3.47-2l-3.26-5.65h-5.55a10.2 10.2 0 01-10.2-10.2V12.79a10.2 10.2 0 0110.2-10.2h24.42a10.2 10.2 0 0110.2 10.2v19a10.2 10.2 0 01-10.2 10.2H31.8l-3.26 5.65a4 4 0 01-3.47 1.95z"
+                                    fill="#703f6c" fill-rule="evenodd" />
+                                <path class="cls-2"
+                                    d="M37.21 5.17a7.64 7.64 0 017.62 7.62v19a7.64 7.64 0 01-7.62 7.62h-6.9l-4 6.93a1.42 1.42 0 01-2.46 0l-4-6.93h-7a7.64 7.64 0 01-7.62-7.62v-19a7.64 7.64 0 017.62-7.62h24.36m0-5.17H12.79A12.81 12.81 0 000 12.79v19a12.81 12.81 0 0012.79 12.74h4.06l2.51 4.35a6.59 6.59 0 0011.42 0l2.51-4.35h3.92A12.81 12.81 0 0050 31.74V12.79A12.81 12.81 0 0037.21 0z" />
+                                <path class="cls-2-heart"
+                                    d="M33.49 17.19a4.32 4.32 0 00-3.57-3c-1.74-.17-3.49.78-4.94 2.64a5.14 5.14 0 00-4.37-2.65 4.61 4.61 0 00-4 3.06c-.94 2.36-.18 5.09 2 7.29l5.29 5.3a1.45 1.45 0 001 .43 1.41 1.41 0 001-.43c.06-.06 3.82-3.8 5.31-5.3 2.3-2.22 3.13-4.98 2.28-7.34z" />
+                            </g>
+                        </g>
+                    </svg></a>
+                <!-- favorite icon -->
+
+                <img src="images/knotless box braids.JPEG" alt=knotless box braids">
```

Comment:
1. This `alt` tag is missing a `"` at its opening.
```suggestion
                <img src="images/knotless box braids.JPEG" alt="knotless box braids">
```
2. Remember to rename your images so their filenames don't have spaces in them and are all lowercase.
3. Consider using your `alt` text to add more context and not only repeat the text already on the page, e.g. "Close up of a woman's scalp as knotless box braids are worked into her hair."

Reply:
Also, this image is over 3000px wide, but your card is 400px wide -- so it is 7.5x larger than it needs to be here. Remember to resize and optimize your images before calling the project done. As you can imagine, on a site with many articles on a single page, those savings would add up, especially for mobile users.